frontend/public/reducers/ui.ts

```diff
@@ -213,11 +214,17 @@ export default (state: UIState, action: UIAction): UIState => {
     case ActionType.ToggleMonitoringGraphs:
       return state.setIn(['monitoring', 'hideGraphs'], !state.getIn(['monitoring', 'hideGraphs']));
 
-    case ActionType.NotificationDrawerToggleExpanded:
+    case ActionType.NotificationDrawerToggleExpanded: {
+      if (window.innerWidth >= breakpointLG) {
```

Comment:
This seems like the wrong place for this logic. The reducer shouldn't know anything about the UI or this event. It also puts the breakpoint check in two different places. (It would be easy to change the breakpoint we use in one place, but not the other.)